### PR TITLE
Use `files` in package.json

### PR DESCRIPTION
**Pull request by SimenB**
Simpler than using `.npmignore`

Before: http://hastebin.com/arocakulox.sh
After: http://hastebin.com/arocakulox.sh

The before is with IntelliJ files as well, but it kinda illustrates the advantage of whitelist vs. blacklist :smile:

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
     }
   ],
   "scripts": {
+    "pretest": "npm run lint",
```
Comment (SimenB) [inline package.json]: Also lint the code when running tests

File: .npmignore [hunk 2]
```diff
@@ -1,12 +0,0 @@
...
-/AUTHORS
-/build
-/test
-/Changelog.txt
-/jsl.conf
-/release.sh
-
-/pkg
-!/pkg/sinon.js
-!/pkg/sinon-ie.js
```
Comment (SimenB) [inline .npmignore]: I don't have ruby installed, and I'd prefer not to. Should `pkg` be included as well?

**Comment by cjohansen** [comment]
This commit includes a bunch of IDEA files. Also, the pretest hook is kinda off-topic, and is in conflict with another PR to be merged.

**Comment by SimenB** [comment]
@cjohansen Yeah I noticed after pushing. I already fixed it.

Want me to strip pretest as well?

**Comment by cjohansen** [comment]
Yup.

**Comment by SimenB** [comment]
Done. What about the `pkg` directory?

**Comment by cjohansen** [comment]
Please keep the PR focused on one thing.

**Comment by SimenB** [comment]
@cjohansen Sure. Is it good now?

**Comment by cjohansen** [comment]
Sure, thanks!

**Pull request merged**